tests/resources/Util.robot

```diff
@@ -75,7 +76,7 @@ Install VIC Appliance To Test Server
 
     # Install the VCH now
     Log To Console  \nInstalling VCH to test server...
-    ${output}=  Run VIC Machine Command  ${certs}  ${vol}  network  'VM Network'
+    ${output}=  Run VIC Machine Command  ${certs}  ${vol}  ${bridge}  ${external}
```

Comment:
we need to actually push this all the way up to line 70 as well, I could have sworn I made that change already but not sure what the heck happened...